recipes/mpitrampoline/bld.bat

```diff
@@ -0,0 +1,10 @@
+cmake -B build -G Ninja ^
```

Comment:
We define an environment variable CMAKE_ARGS on all platforms. Please use it. It may help you to find the fortran compiler on Windows.

https://conda-forge.org/docs/maintainer/knowledge_base/#using-cmake